files/src/ui/components/__component__/component-test.ts

```diff
@@ -0,0 +1,13 @@
+import { setupRenderingTest } from '@glimmer/test-helpers';
+import hbs from '@glimmer/inline-precompile';
+
+const { module, test } = QUnit;
+
+module('Component: <%= component %>', function(hooks) {
+  setupRenderingTest(hooks);
```

Comment:
This has been merged already, but `hooks` will be undefined, so this blueprint is going to generate a test that has a syntax error.